cli/fs_util.rs

```diff
@@ -90,17 +90,30 @@ pub fn resolve_from_cwd(path: &Path) -> Result<PathBuf, AnyError> {
 
 /// Checks if the path has extension Deno supports.
 pub fn is_supported_ext(path: &Path) -> bool {
-  let lowercase_ext = path
-    .extension()
-    .and_then(|e| e.to_str())
-    .map(|e| e.to_lowercase());
-  if let Some(ext) = lowercase_ext {
-    ext == "ts" || ext == "tsx" || ext == "js" || ext == "jsx" || ext == "mjs"
+  if let Some(ext) = get_extension(path) {
+    matches!(ext.as_str(), "ts" | "tsx" | "js" | "jsx" | "mjs")
+  } else {
+    false
+  }
+}
+
+/// This function is similar to is_supported_ext but also allows .md extension.
+pub fn is_supported_ext_md(path: &Path) -> bool {
```

Comment:
I choose to duplicate the function (also results in duplicate test cases) instead of calling `is_supported_ext` for the rest of the extensions because I thought the latter would result in unnecessary function calls.